packages/suite-storage/.eslintignore

```diff
@@ -0,0 +1,35 @@
+# general
```

Comment:
do we need to have this file in every package? wouldn't it be better to have it globally?

Reply:
I think we do need it. We have added it to every package (https://github.com/trezor/trezor-suite/pull/401). @vladimirvolek could know something about it.